samples/net/lldp/src/main.c

```diff
@@ -7,9 +7,9 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(net_lldp_sample, LOG_LEVEL_DBG);
 
-#include <zephyr/zephyr.h>
+#include <zephyr/kernel.h>
 
-#include <zephyr/zephyr.h>
+#include <zephyr/kernel.h>
```

Comment:
It is included twice in this file, could remove one.

Reply:
fixed